### ISSUE TITLE
R14 Memory error

### ISSUE THREAD

**Issue by psychobmw98** [BochilGaming/games-wabot#331]
This system use up more memory on heroku and the it crashes. Can you fix this please

**Comment by DineshValor** [comment]
Is this issue fixed?

**Comment by BochilGaming** [comment]
only reduce memory usage maybe?

**Comment by DineshValor** [comment]
> only reduce memory usage maybe?

Yep, thanx, well i was shifted to replit and its more efficient and stable than heroku.
